.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -175,15 +175,50 @@ AddType text/vtt                            vtt
     # Redirect for renamed external DNS page
     RewriteRule ^container-platform/4\.10/networking/external_dns_operator/nw-installing-external-dns-operator.html /container-platform/4.10/networking/external_dns_operator/nw-installing-external-dns-operator-on-cloud-providers.html [NE,R=301]
 
-   # Redirect for renamed Ingress Controller document
+    # Redirect for renamed Ingress Controller document
     RewriteRule ^container-platform/(4\.1[2-6])/networking/nw-ingress-controller-endpoint-publishing-strategies\.html$ /container-platform/$1/networking/nw-configuring-ingress-controller-endpoint-publishing-strategy.html [NE,R=302,L]
 
     # Redirect for Networking operators move per https://github.com/openshift/openshift-docs/pull/85085
     RewriteRule ^container-platform/(4\.1[6-8])/networking/aws_load_balancer_operator/(.*)? /container-platform/$1/networking/networking_operators/aws_load_balancer_operator/$2 [NE,R=302]
+
+    RewriteRule rosa/networking/aws-load-balancer-operator.html /container-platform/$1/networking/networking_operators/aws-load-balancer-operator.html [NE,R=302]
```

Comment:
Hi @bscott-rh . Is this syntax correct? Rosa redirects are also needed :grimacing: .